### ISSUE TITLE
Can only use custom script when install: true

### ISSUE THREAD

**Issue by PirateGrunt** [travis-ci/travis-ci#7047]
I'm trying to use Travis on an R project, which will not produce a package. I've so indicated in travis.yml as:

    script: make

However, I get the following error during the build:

    No DESCRIPTION file found, user must supply their own install and script steps

I've done so and the build is fine now. However, this behavior suggests that `R CMD` is being called during the `install` step. I've searched the docs and can't confirm this. Can anyone comment? I would expect `R CMD` to be housed in the `script` step. That would keep installation clean and consistent while enabling R builds that aren't meant to produce a package.

**Comment by jimhester** [comment]
The package (or script) dependencies are installed in the `install` step based on the DESCRIPTION file, the check is done in the `script` step.

**Comment by PirateGrunt** [comment]
Thanks so much. This resolves my issue.

Hate to ask, but is there any way this could be added to the (otherwise excellent) documentation?

**Comment by BanzaiMan** [comment]
Indeed, our R documentation lacks the "Default Test Script" section. ([Python](https://docs.travis-ci.com/user/languages/python#Specifying-Test-Script), [Ruby](https://docs.travis-ci.com/user/languages/ruby#Default-Test-Script), [JavaScript](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Default-Test-Script))
